docs/guide/tutorials.rst

```diff
@@ -355,6 +357,134 @@ Targeted flux analysis with GC-MS SIM Agilent
 Non-targeted FIA-MS analysis with Thermo Orbitrap
 -------------------------------------------------
 
+This tutorial walks you through the workflow for analyzing non-targeted FIA-MS 
+data starting from input file generation, to processing the data in SmartPeak, 
+to reviewing the data in SmartPeak, to reporting the results for later use.
+
+Objectives
+~~~~~~~~~~
+
+#. Obtaining the SOP for the workflow.
+#. Choosing a data set for demonstrating the workflow.
+#. Creating an optimized SmartPeak input templates for running the workflow.
+
+The Workflows include
+~~~~~~~~~~~~~~~~~~~~~
+
+#. Processing Unknowns
```

Comment:
If you can modify the workflow to include the following, that would be great:
1. Defining the accurate mass search database
2. Processing Unknowns
3. Reviewing the results